verification/serverless.yml

```diff
@@ -91,29 +91,29 @@ functions:
               - X-Amz-Security-Token
               - X-Amz-User-Agent
 
-  submit:
-    handler: verification/application/handlers/jumio_verification_handlers.submit
-    role: ${file(./config.${self:provider.stage}.json):ROLE}
-    vpc:
-      securityGroupIds:
-        - ${file(./config.${self:provider.stage}.json):SG1}
-        - ${file(./config.${self:provider.stage}.json):SG2}
-      subnetIds:
-        - ${file(./config.${self:provider.stage}.json):VPC1}
-        - ${file(./config.${self:provider.stage}.json):VPC2}
-    events:
-      - http:
-          method: GET
-          path: /{verification_id}/submit
-          cors:
-            origin: ${file(./config.${self:provider.stage}.json):ORIGIN}
-            header:
-              - Content-Type
-              - X-Amz-Date
-              - Authorization
-              - X-Api-Key
-              - X-Amz-Security-Token
-              - X-Amz-User-Agent
+  #  submit:
```

Comment:
Why commented code  ?

Reply:
we testing it right now in AWS API gateway